compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -1024,6 +1024,74 @@ struct QuantizeWeights final : public luci::CircleNodeMutableVisitor<bool>
   bool visit(luci::CircleNode *) { return false; }
 };
 
+/** EXAMPLE
+ *
+ * BEFORE
+ *
+ *         [CircleNode]       [CircleConst]
+ *           (qparam1)           (FP32)
+ *                   \            /
+ *                    \          /
+ *                    [CirclePack]
+ *                     (qparam2)
+ *
+ *  AFTER
+ *
+ *         [CircleNode]        [CircleConst]   [CircleConst] <- Dead node
+ *           (qparam2)           (qparam2)         (FP32)
+ *                   \            /
+ *                    \          /
+ *                    [CirclePack]
+ *                     (qparam2)
```

Comment:
(optional) there is `qparam1` and `qparam2` which I could not find about this in this PR.
it would help understand if there is some notes about this.